File: Assets/AltUnityTester/AltUnityDriver/Commands/OldFindObjects/FindElementWhereNameContains.cs
public class FindElementWhereNameContains : CommandReturningAltElement
{
    string name;
    string cameraName;
    bool enabled;
    public FindElementWhereNameContains(SocketSettings socketSettings, string name, string cameraName, bool enabled) : base(socketSettings)
    {
        this.name = name;
        this.cameraName = cameraName;
        this.enabled = enabled;
    }
    public AltUnityObject Execute()
    {
        Socket.Client.Send(toBytes(CreateCommand("findObjectWhereNameContains", name, cameraName, enabled.ToString())));
        return ReceiveAltUnityObject();
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/OldFindObjects/find_element_where_name_contains.py
from altunityrunner.commands.command_returning_alt_elements import CommandReturningAltElements
class FindElementWhereNameContains(CommandReturningAltElements):
    def __init__(self, socket,request_separator,request_end,appium_driver,value,camera_name,enabled):
        super().__init__(socket,request_separator,request_end,appium_driver)
        self.value=value
        self.camera_name=camera_name
        self.enabled=enabled
    
    def execute(self):
        if self.enabled==True:
            data = self.send_data(self.create_command('findObjectWhereNameContains', self.value , self.camera_name,'true' ))
        else:
            data = self.send_data(self.create_command('findObjectWhereNameContains', self.value , self.camera_name,'false' ))
        return self.get_alt_element(data)